CMakeLists.txt

```diff
@@ -33,13 +63,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 ")
 include_directories(
   include
   ${catkin_INCLUDE_DIRS}
-  ${OPENVR}/headers
 )
 
 # VIVE node:
 add_executable(vive_node 
   src/vive_node.cpp
   src/vr_interface.cpp
+  ${OPENVR_SAMPLE_UTILS}
+)
+target_link_libraries(vive_node
+  ${catkin_LIBRARIES}
+  ${OPENVR_LIB}
+  openvr_api
+  ${GLEW_LIBRARIES}
+  ${GTK2_LIBRARIES}
+  ${SDL2_LIBRARY}
+  ${OpenCV_LIBS}
```

Comment:
if you add `find_package(catkin REQUIRED cv_bridge)` then opencv libraries should provided by `${catkin_LIBRARIES}` , please check that